Sources/CorePayments/Networking/GraphQL/GraphQLClient.swift

```diff
@@ -23,13 +25,20 @@ class GraphQLClient {
         let decoded: GraphQLQueryResponse<T> = try parse(data: data)
         return decoded
     }
-
+    
     func parse<T: Decodable>(data: Data) throws -> T {
         return try jsonDecoder.decode(T.self, from: data)
     }
 
-    func createURLRequest(requestBody: Data) throws -> URLRequest {
-        var urlRequest = URLRequest(url: environment.graphQLURL)
+    func createURLRequest(name: String? = nil, requestBody: Data) throws -> URLRequest {
+        var urlString = environment.graphQLURL.absoluteString
+        if let name {
+            urlString.append("?\(name)")
+        }
+        guard let url = URL(string: urlString) else {
+            throw GraphQLError(message: "error fetching url", extensions: nil)
```

Comment:
Can we default `extensions` to nil in `GraphQLError` so we don't need to include it here?

Reply:
217c6c5049585920865152ece58a51a782a2cbb5